move/its/sources/events.move

```diff
@@ -128,3 +128,50 @@ public(package) fun unregistered_coin_received<T>(
         decimals,
     });
 }
+ 
+// ---------
+// Test Only 
+// ---------
+#[test_only]
+use its::coin::COIN;
+#[test_only]
+use its::token_id;
```

Comment:
```suggestion
#[test_only] {
    use its::coin::COIN;
    use its::token_id;
}
```
i wonder if Move supports this similar to Rust

Reply:
it does not